index.js

```diff
@@ -4,20 +4,23 @@
   /**
    * This function receives a string input and converts it to mOcKiNgCaSe.
    * @param {string} input - string to be converted
-   * @param {object} [options={random: false}] - options for converting
+   * @param {object} [options={random: false, firstUpper: false}] - options for converting
    * @param {boolean} options.random=false - using random for converting
+   * @param {boolean} options.firstUpper=false - use to capitalize all even indices
    * @returns {string} string in mOcKiNgCaSe
    */
   function mOcKiNgCaSe(input, options) {
     options = Object.assign({
-      random: false
+      random: false,
+      firstUpper: false,
     }, options);
 
     return input.replace(/./g, (s, i) => {
       if (options.random) return Math.round(Math.random()) ? s.toUpperCase() : s.toLowerCase()
```

Comment:
What do you think about this solution?
```suggestion
       if (options.random) {
        if (options.firstUpper && i === 0) return s.toUpperCase();
        return Math.round(Math.random()) ? s.toUpperCase() : s.toLowerCase();
      }
```

Reply:
and possible test:
```js
const input = 'hello world';
const expectedRegex = /^[A-Z].*/g;
const options = {
  random: true,
  firstUpper: true,
};
expect(mOcKiNgCaSe(input, options)).toMatch(expectedRegex);
```